# Basic Statistics_Set3 (Q5)

In January 2005, a company that monitors Internet traffic (WebSideStory) reported that its sampling revealed that the Mozilla Firefox browser launched in 2004 had grabbed a 4.6% share of the market.

I.	If the sample were based on 2,000 users, could Microsoft conclude that Mozilla has a less than 5% share of the market?

II.	WebSideStory claims that its sample includes all the daily Internet users. If that’s the case, then can Microsoft conclude that Mozilla has a less than 5% share of the market?



In [1]:
import numpy as np
from scipy import stats
from scipy.stats import norm


I. If the sample were based on 2,000 users, could Microsoft conclude that Mozilla has a less than 5% share of the market? <br>
Now suppose the population proportion share of market by the Mozilla is p = 0.5 then sample proportion share of market by the Mozilla is p̂ = 0.46 <br>
Assume Null Hypothesis as Ho: p >= 5% market share Thus Alternate Hypothesis as Ha: p < 5% market share

Ho: p>=5% <br>
H1: p<=5% <br>
Apply One-Sample One-Tail z-test

calculate z-score or value<br>
z_scores = (p̂-p)/√(p(1-p)/n) ...N(0,1)


In [2]:
z_scores=(0.046-0.05)/(np.sqrt((0.05*(1-0.05))/2000))
z_scores

-0.820782681668124

 Find Probability assuming null hyposthesis, so as to compare with Type-1 error α = 0.05


In [3]:
p_value=1-stats.norm.cdf(abs(z_scores))
p_value

0.20588503245107104

The p value for z= -0.82 is p = 0.20 which is greater than critical value(0.05) and thus we cannot reject null hypothesis. <br>
Thus we conclude that Mozilla has share greater than 5%